client/tools/mgr-osad/src/osa_dispatcher.py

```diff
@@ -17,7 +17,10 @@
 import select
 import socket
 import string
-import SocketServer
+try: # python 3
+    import sockerserver
```

Comment:
Looks fishy... should that be `socketserver` with a 't'?

Reply:
I think so: https://docs.python.org/3/library/socketserver.html